recorder/src/main/resources/tracking.js

```diff
@@ -73,6 +73,7 @@ jsflight.TrackHash = function(event) {
         console.log(e);
     } finally {
         jsflight.eventId++;
+        jsflight.rebindAllHandlers();
```

Comment:
But you are rebinding immediately after an event, and at this very monent there might be no dom modifications at all.
In that case I think it is better to listen to dom modification event, and process newly appeared iframes, its'n it?